packages/docusaurus-theme-classic/codeTranslations/tr.json

```diff
@@ -0,0 +1,68 @@
+  "theme.SearchPage.emptyResultsTitle": "Dokümanlarda ara",
+  "theme.SearchPage.existingResultsTitle": "Arama sonuçları",
+  "theme.SearchPage.fetchingNewResults": "Yeni sonuçlar getiriliyor...",
+  "theme.SearchPage.inputLabel": "Ara",
+  "theme.SearchPage.inputPlaceholder": "Aramanızı buraya yazın",
+  "theme.SearchPage.noResultsText": "Hiçbir sonuç bulunamadı",
+  "theme.blog.paginator.navAriaLabel": "Blog gönderi sayfası navigasyonu",
+  "theme.blog.paginator.newerEntries": "Yeni Girdiler",
+  "theme.blog.paginator.olderEntries": "Eski Girdiler",
+  "theme.blog.post.date": "{day} {month} {year}",
+  "theme.blog.post.nPosts": "{count} gönderi",
+  "theme.blog.post.onePost": "Bir gönderi",
+  "theme.blog.post.paginator.navAriaLabel": "Blog gönderi sayfası navigasyonu",
+  "theme.blog.post.paginator.newerPost": "Daha Yeni Gönderi",
+  "theme.blog.post.paginator.olderPost": "Daha Eski Gönderi",
+  "theme.blog.post.readMore": "Daha Fazla",
+  "theme.blog.post.readingTime": "{readingTime} dakikalık okuma",
+  "theme.blog.tagTitle": "\"{tagName}\" ile etiketlenmiş {nPosts}",
+  "theme.common.editThisPage": "Bu sayfayı düzenle",
+  "theme.common.headingLinkTitle": "Başlığa doğrudan bağlantı",
+  "theme.common.month.april": "Nisan",
+  "theme.common.month.august": "Ağustos",
+  "theme.common.month.december": "Aralık",
+  "theme.common.month.february": "Şubat",
+  "theme.common.month.january": "Ocak",
+  "theme.common.month.july": "Temmuz",
+  "theme.common.month.june": "Haziran",
+  "theme.common.month.march": "Mart",
+  "theme.common.month.may": "Mayıs",
+  "theme.common.month.november": "Kasım",
+  "theme.common.month.october": "Ekim",
+  "theme.common.month.september": "Eylül",
+  "theme.common.skipToMainContent": "Ana içeriğe geç",
+  "theme.docs.paginator.navAriaLabel": "Dokümanlar sayfası navigasyonu",
+  "theme.docs.paginator.next": "Sonraki",
+  "theme.docs.paginator.previous": "Önceki",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Kenar çubuğunu daralt",
+  "theme.docs.sidebar.collapseButtonTitle": "Kenar çubuğunu daralt",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Kenar çubuğunu genişlet",
+  "theme.docs.sidebar.expandButtonTitle": "Kenar çubuğunu genişlet",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "Menüyü kapat",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "Menüyü aç",
+  "theme.docs.versions.latestVersionLinkLabel": "en son sürüm",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Güncel belgeler için bkz. {latestVersionLink} ({versionLabel}).",
+  "theme.docs.versions.unmaintainedVersionLabel": "Bu, {siteTitle} {versionLabel} dokümantasyonudur ve bakımı sonlanmıştır.",
+  "theme.docs.versions.unreleasedVersionLabel": "Bu, {siteTitle} {versionLabel} sürümü için yayınlanmamış dokümantasyondur.",
+  "theme.lastUpdated.atDate": "{date}",
+  "theme.lastUpdated.byUser": "{user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "En son {atDate} tarihinde {byUser} tarafından güncellendi",
```

Comment:
I think this won't work in all cases.

https://v2.docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs

![image](https://user-images.githubusercontent.com/749374/109540642-11c0ee00-7ac3-11eb-9070-8fb426f4c533.png)


We have to handle those 3 cases:
- Last updated by Sébastien
- Last updated on Monday
- Last updated on Monday by Sébastien

Let's imagine user does not want to display the update date, you would end up with a string like `En son tarihinde Sébastien tarafından güncellendi`.

I can't be 100% sure as I'm not speaking Turkish but I feel it's not correct right ? :)

Reply:
Oh, I see. You're right, it won't handle the first two cases correctly. Let me fix it. :)